src/integrations/gitlens/gitlens.d.ts

```diff
@@ -0,0 +1,55 @@
+'use strict';
```

Comment:
microsoft header?

Reply:
Should it have one? As its from GitLens not MS